README.md

```diff
@@ -1305,6 +1306,18 @@ To configure the AI agent for your dashboard, you need to add the `agent` config
 
 The agent will use the configured models to process natural language commands and perform database operations using the master key from your app configuration.
 
+### Environment Variables
```

Comment:
No extra chapter for this. Add a column in the table, or add it to the main table for env vars above.